assignment-2/src/model/model.js

```diff
@@ -0,0 +1,90 @@
+import { EventEmitter } from "events";
+
+class Model extends EventEmitter {
+
+    constructor() {
+        super();
+        this.state = {
+            questionid: 2,
+            searchWord: "",
+            questions: [{
+                id: 1,
+                title: "Java 8",
+                body: "What are the new features in Java8?",
+                tags: "java,programming",
+                author: "John",
+                postDate: "4/13/2019 3:26 PM"
+            }, {
+                id: 2,
+                title: "Static",
+                body: "Difference between static in Java and C++?",
+                tags: "java,C++",
+                author: "Kate",
+                postDate: "4/13/2019 3:48 PM"
+            }],
+            newQuestion: {
+                title: "",
+                body: "",
+                tags:""
+            }
+        };
+    }
+
+
+    addQuestion(title, body, tags) {
+
+        // get current date
```

Comment:
I would recommend that you use `momentjs` for formatting dates (you have to install it with `npm i -S momentjs`). 

If you really want to do this by yourself, extract it at least in a helper function.